### DISCUSSION TITLE
Link auf Planzeichenverordnung oder andere Info

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#84] [Q&A]
Bei uns ist die Frage aufgetaucht, ob es neben dem Link zu den Metadaten, den wir allerdings nicht für etwas anderes Missbrauchen wollen

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/90205b82-58e8-46fd-b01d-0dd4a4925572)

 eine Möglichkeit gibt oder geben wird, für ein bestimmtes Thema oder eine Themengruppe einen "Info-Button" zu erstellen über den man z.B. auf eine Planzeichenverordnung oder einen anderen Hinweis im PDF Format verlinken kann.
Wir hatten das in WebGIS4 z.B. bei den Gefahrenzonen und beim Flächenwidmungsplan verwendet.

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/1b39a7cb-2eca-4e8c-8ed4-2b9158608d54)

**Comment by gis-eni** [comment]
War das im WebGIS 5 nicht das gleich. Auch dort war das ``i`` der Metadaten Button?
